### DISCUSSION TITLE
[companion] Remind me later about this article

### DISCUSSION THREAD

**Discussion by GrahamTheDev** [dailydotdev/daily#608] [Feature Requests]
Option to make an article as "read later".

In X days it shows up in the feed with a special icon and doesn't go away until dismissed / read.

**Comment by idoshamun** [comment]
Hi @GrahamTheDev,

I think this is a platform-wide feature that we should add.
There's a conversation going on here: https://github.com/dailydotdev/daily/discussions/364
